rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestControllerHelper.java

```diff
@@ -1655,4 +1700,357 @@ public ClearingRequest updateCRSize(ClearingRequest clearingRequest, Project pro
         }
         return clearingRequestService.getClearingRequestById(clearingRequest.getId(), sw360User);
     }
+
+    public SPDXDocument generateSpdxDocument() {
```

Comment:
These generators are defined as `public static` in `SpdxDocumentDatabaseHandler`. Can't we reuse them?

Or move them to a util class if backend is not a dependency of Rest module.

Reply:
I moved them to SW360Utils